### PR TITLE
Add tests on sending invalid parcels

### PR DESCRIPTION
**Pull request by jhs7jhs**

### CODE REVIEW

File: test/src/integration/basic.test.ts [hunk 1]
```diff
@@ -63,6 +105,268 @@ describe("solo - 1 node", () => {
...
+
+        test.each(["0x01" + "0".repeat(64), "0x" + "f".repeat(128)])(
+            "nonce: %p",
+            async (nonce, done) => {
+                parcelEncoded[0] = nonce;
+                try {
+                    await node.sendSignedParcelWithRlpBytes(
+                        RLP.encode(parcelEncoded)
+                    );
+                    done.fail();
```
Comment (joojis) [inline test/src/integration/basic.test.ts]: It's a common practice to return here.

return done.fail()

File: test/src/integration/basic.test.ts [hunk 2]
```diff
@@ -63,6 +105,268 @@ describe("solo - 1 node", () => {
...
+                } catch (e) {
+                    expect(e).toEqual(ERROR.INVALID_RLP_INVALID_LENGTH);
+                }
+                done();
+            }
+        );
+
```
Comment (joojis) [inline test/src/integration/basic.test.ts]: Could you add an INVALID_NETWORK_ID case? Any 2 characters string except for "tc" will do.

File: test/src/integration/basic.test.ts (hunk 1 above)
Comment (jhs7jhs) [reply test/src/integration/basic.test.ts]: Moved `done()` into the `catch`.

File: test/src/integration/basic.test.ts (hunk 2 above)
Comment (jhs7jhs) [reply test/src/integration/basic.test.ts]: Added.

**Comment by jhs7jhs** [comment]
I'm still not sure that it is appropriate to add these test cases into `basic.test.ts`.

**Comment by joojis** [comment]
@jhs7jhs How about `verification`?

**Comment by jhs7jhs** [comment]
@joojis It sounds good. I'll move it.

**Pull request merged**